docs/_content/ontwikkelaars/algemeen/archiveren.md

```diff
@@ -104,11 +193,9 @@ Het vernietigen is het definitief verwijderen van data volgens de NEN2082 (Eis 8
    * `Objecttype` (*optioneel*) Het soort object in de registratie dat het procesobject representeert.
    * `Datumkenmerk` (*optioneel*) Naam van de attribuutsoort van het procesobject dat bepalend is voor het einde van de procestermijn. 
 
-
-
 TODO:
 
-* `Zaak.StartdatumBewaartermijn` (nieuw!) De datum die de start markeert van de termijn waarop het zaakdossier vernietigd moet worden. 
+* `Zaak.StartdatumBewaartermijn` (nieuw!) De datum die de start markeert van de termijn waarop het zaakdossier vernietigd moet worden.
```

Comment:
Deze begrijp ik niet meteen. Los van hoe het in de praktijk gaat is vernietigen toch een atomaire gebeurtenis? Hoe verhoudt zich dat tot een begrip als 'termijn'. en hoe verhoudt dit zich tot `Archiefactiedatum`? Dat is voor zaken met `Archiefnominatie` vernietigen toch het moment waarop daadwerkelijk vernietigd moet worden.

Reply:
Volgens mij is dit hetzelfde als brondatum, maar had @joeribekker een reden om deze toe te voegen.

Reply:
Dit komt niet van mij he? Deze stond in RGBZ 2.0.2.

Hij is in TODO blijven staan omdat we em niet gebruikt hebben bij het Archiveren epic.

Reply:
Dit attribuut <code>StartdatumBewaartermijn</code> weglaten uit de documentatie.